src/components/fields/SSICheckbox/index.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { FC } from 'react'
+import { TouchableOpacity, ViewStyle } from 'react-native'
+
+import { HIT_SLOP_DISTANCE } from '../../../@config/constants'
+import {
+  SSICheckboxSelectedContainerStyled as SelectedContainer,
+  SSICheckboxUnselectedContainerStyled as UnselectedContainer
+} from '../../../styles/components'
+
+export interface IProps {
+  onValueChange?: () => Promise<void>
+  defaultValue?: boolean
```

Comment:
Maybe initialValue makes a bit more sense. That really makes clear the value is only used during init

Reply:
changed to initialValue